mobile/src/main/res/drawable/ic_none.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp"
+    android:viewportHeight="24.0" android:viewportWidth="24.0"
+    android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:pathData=""/>
```

Comment:
```
01-31 11:14:19.230  6991  6991 E AndroidRuntime: FATAL EXCEPTION: main
01-31 11:14:19.230  6991  6991 E AndroidRuntime: Process: org.openhab.habdroid.beta, PID: 6991
01-31 11:14:19.230  6991  6991 E AndroidRuntime: android.content.res.Resources$NotFoundException: Drawable org.openhab.habdroid.beta:drawable/ic_none with resource ID #0x7f08009d
01-31 11:14:19.230  6991  6991 E AndroidRuntime: Caused by: android.content.res.Resources$NotFoundException: File res/drawable-anydpi-v21/ic_none.xml from drawable resource ID #0x7f08009d
01-31 11:14:19.230  6991  6991 E AndroidRuntime:        at android.content.res.ResourcesImpl.loadDrawableForCookie(ResourcesImpl.java:725)
[...]
01-31 11:14:19.230  6991  6991 E AndroidRuntime: Caused by: java.lang.IllegalArgumentException: Path string cannot be empty.
01-31 11:14:19.230  6991  6991 E AndroidRuntime:        at android.util.PathParser.nCreatePathDataFromString(Native Method)
01-31 11:14:19.230  6991  6991 E AndroidRuntime:        at android.util.PathParser.-wrap1(PathParser.java)
01-31 11:14:19.230  6991  6991 E AndroidRuntime:        at android.util.PathParser$PathData.<init>(PathParser.java:74)
01-31 11:14:19.230  6991  6991 E AndroidRuntime:        at android.graphics.drawable.VectorDrawable$VFullPath.updateStateFromTypedArray(VectorDrawable.java:1556)
[...]
```

Reply:
Fixed by adding an icon. I guess some prefs look better without an icon, so the icons can point the user to the important prefs. Using `android:iconSpaceReserved="true"` didn't work for me, I guess because we don't use PreferenceFragmentCompat. But changing that is out of scope of this PR.